examples/03_additive_thermal_history.py

```diff
@@ -49,15 +49,13 @@
 additive = Additive()
 
 ###############################################################################
-# Select model
-# ------------
-# Select the geometry model. Currently, PyAdditive supports
-# two types of geometry specifications,
-# :class:`STLFile <ansys.additive.core.geometry_file.StlFile>` and
-# :class:`BuildFile <ansys.additive.core.geometry_file.BuildFile>`.
+# Specify model
+# -------------
+# Specify the geometry model. Currently, PyAdditive supports two types of geometry
+# specifications, :class:`StlFile` and :class:`BuildFile`.
```

Comment:
Is there a way to add a noun, something like this: ... PyAdditive supports two types of geometry specifications,
the :class:`StlFile` class and the :class:`BuildFile` class. Also, style guide says to write timeless documentation, avoiding words like currently. The doc should always reflect the current state of the code, so it is unneeded. I've not made these changes because I'll let you decide!